packages/cli/src/utils/changelog.js

```diff
@@ -34,16 +80,37 @@ const getChangelogFromMarkdown = (version, markdown) => {
   endingLine -= 1;
 
   const changelogLines = lines.slice(startingLine, endingLine);
+  const changelog = [];
+  const appMetadata = [];
+  const issueMetadata = [];
+
+  for (const line of changelogLines) {
+    let changeType = null;
+    if (startsWithIgnoringSymbols(line, 'update')) {
+      changeType = 'FEATURE_UPDATE';
+    } else if (startsWithIgnoringSymbols(line, 'fix')) {
+      changeType = 'BUGFIX';
+    }
+    if (changeType) {
+      appMetadata.push(...getAppMetadata(line, changeType));
+      issueMetadata.push(...getIssueMetadata(line, changeType));
+    }
+    changelog.push(line);
+  }
 
-  return changelogLines.join('\n');
+  return {
+    changelog: changelog.join('\n'),
+    appMetadata: absentIfEmpty(appMetadata),
+    issueMetadata: absentIfEmpty(issueMetadata),
+  };
```

Comment:
While we're here, I'd like to address [PDE-3820](https://zapierorg.atlassian.net/browse/PDE-3820) as well to improve the markdown parsing. Hopefully this is not too much.

Here's what I came up with:

```javascript
const getChangelogFromMarkdown = (version, markdown) => {
  const lines = markdown
    .replace(/\r\n/g, '\n')
    .replace(/\r/g, '\n')
    .split('\n');

  let startingLine = lines.findIndex((line) =>
    RegExp(`^#{1,4} .*${version.replaceAll('.', '\\.')}`).test(line)
  );

  if (startingLine === -1) {
    throw new Error(`Version '${version}' not found in changelog`);
  }

  startingLine++;

  // Find the next line that starts with one or more '#' chars
  let endingLine = lines
    .slice(startingLine)
    .findIndex((line) => /^#{1,4} /.test(line));

  if (endingLine === -1) {
    endingLine = lines.length;
  }
  endingLine += startingLine;

  const changelogLines = lines.slice(startingLine, endingLine);
  const changelog = [];
  const appMetadata = [];
  const issueMetadata = [];

  for (const line of changelogLines) {
    let changeType = null;
    if (startsWithIgnoringSymbols(line, 'update')) {
      changeType = 'FEATURE_UPDATE';
    } else if (startsWithIgnoringSymbols(line, 'fix')) {
      changeType = 'BUGFIX';
    }
    if (changeType) {
      appMetadata.push(...getAppMetadata(line, changeType));
      issueMetadata.push(...getIssueMetadata(line, changeType));
    }
    changelog.push(line);
  }

  return {
    changelog: changelog.join('\n').trim(),
    appMetadata: absentIfEmpty(appMetadata),
    issueMetadata: absentIfEmpty(issueMetadata),
  };
};
```

Can you use this new `getChangelogFromMarkdown` function in this PR? Thanks!